src/NServiceBus.AcceptanceTests/Recoverability/When_message_is_moved_to_error_queue.cs

```diff
@@ -54,6 +54,30 @@ public Task May_send_outgoing_messages(TransportTransactionMode transactionMode)
             .Run();
         }
 
+        [TestCase(TransportTransactionMode.SendsAtomicWithReceive)]
+        [TestCase(TransportTransactionMode.TransactionScope)]
+        [TestCase(TransportTransactionMode.ReceiveOnly)]
+        [TestCase(TransportTransactionMode.None)]
```

Comment:
I'm not sure I follow what you're saying here. If you didn't test multiple transaction modes, what would the test need to do differently?